README.md

```diff
@@ -109,15 +109,25 @@ For Jest debugging guide using Node.js, see [docs/tests/jest.md](docs/tests/jest
 
 ### Running E2E Tests
 
-Our e2e test suite can be run on either Firefox or Chrome.
+Our e2e test suite can be run on either Firefox or Chrome. Here's how to get started with e2e testing:
 
-1. **required** `yarn build:test` to create a test build.
-2. run tests, targeting the browser:
+#### Preparing a Test Build
 
-- Firefox e2e tests can be run with `yarn test:e2e:firefox`.
-- Chrome e2e tests can be run with `yarn test:e2e:chrome`. The `chromedriver` package major version must match the major version of your local Chrome installation. If they don't match, update whichever is behind before running Chrome e2e tests.
+Before running e2e tests, you'll need a test build. You have two options:
 
-These test scripts all support additional options, which might be helpful for debugging. Run the script with the flag `--help` to see all options.
+1. Use `yarn download-builds:test` to quickly download and unzip test builds for Chrome and Firefox into the `./dist/` folder.
+2. Create a custom test build: for testing against different build types, use `yarn build:test`. This command allows you to generate test builds for various types, including:
+    - `yarn build:test` for main build
+    - `yarn build:test:flask` for flask build
+    - `yarn build:test:mmi` for mmi build
+    - `yarn build:test:mv3` for mv3 build
+
```

Comment:
should we mentioned a 3rd option too: `yarn start:test` ?

Reply:
Thanks Mariona, great catch, i have added it. Please check.